### PR TITLE
text update

### PR DESCRIPTION
**Pull request by danieledebernardinDTD**
## Description

As already mentioned in #ISSUE_NUMBER, this PR tackles:
* ...
* ...
* ...

In particular, the ...

## Checklist

- [ ] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [ ] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Comment by danieledebernardinDTD** [comment]
@sebbalex quando puoi, grazie come sempre

**Comment by danieledebernardinDTD** [comment]
@sebbalex nel frattempo ho aggiunto dei pezzi vari, l'ultimo mi dà errore
vedi tu cosa ho sbagliato :(

File: contents/home-page/home.yml [hunk 1]
```diff
@@ -124,42 +124,43 @@ opportunity:
       id: centri-facili
 heroCarouselNews:
   - id: 1
+    category: Novità
+    title: "Un nuovo canale diretto per i fornitori"
+    description: "Un Help Desk dedicato per rispondere ai quesiti dal mondo ICT sugli Avvisi"
+    source: 'innovazione.gov.it'
+    linkTo: 'https://innovazione.gov.it/notizie/articoli/pa-digitale-2026-un-nuovo-canale-diretto-per-i-fornitori/'
+   - id: 2
```
Comment (sebbalex) [inline contents/home-page/home.yml:132]:
```suggestion
  - id: 2
```

**Review by sebbalex — APPROVED** [review]

**Review by sebbalex — COMMENTED** [review]
fix: lint

File: src/layouts/AssistenzaPage.jsx [hunk 2]
```diff
@@ -573,8 +573,8 @@ export const AssistenzaPage = () => {
           <Col xs={12} md={6} lg={5}>
             <h1 className={classes.titleUpdate}>Assistenza</h1>
             <div className={classes.subtitleUpdate}>
-              Un team dedicato è a tua disposizione per chiarire dubbi e approfondire temi di interesse. Compila il
-              modulo sottostante e invia la richiesta.
+              Un team dedicato è a disposizione di PA locali e fornitori per chiarire dubbi e approfondire temi di
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:576]:
```suggestion
              Un team dedicato è a disposizione di PA locali e fornitori per chiarire dubbi e approfondire temi di
```

**Pull request merged**